src/main/java/hudson/plugins/jira/JiraChangeLogAnnotator.java

```diff
@@ -26,13 +27,36 @@
  */
 @Extension
 public class JiraChangeLogAnnotator extends ChangeLogAnnotator {
+	
     private static final Logger LOGGER = Logger.getLogger(JiraChangeLogAnnotator.class.getName());
 
+    public JiraChangeLogAnnotator()
+    {
+        super();
+        if(LOGGER.isLoggable(Level.FINE))
+        {
+        	LOGGER.info("JiraChangeLogAnnotator registered");
```

Comment:
LOGGER.log(Level.FINE, "JiraChangeLogAnnotator registered")
BTW it does not guaranteed that annotator is registered. You would have to check Jenkins' extension lists